target/linux/ramips/dts/mt7621_netgear_r.dtsi

```diff
@@ -90,18 +64,18 @@
 
 &pcie0 {
 	wifi@0,0 {
-		compatible = "pci14c3,7662";
-		reg = <0x0000 0 0 0 0>;
+		compatible = "mediatek,mt76";
```

Comment:
Can anyone confirm that the generic compatible will work here?

Reply:
@981213 Can you comment on this one?
Some devices have "pci14c3,7662", some have "mediatek,mt76". Can this be merged  as "mediatek,mt76" or do I have to keep separate compatibles?